### PR TITLE
Add HTML structure and first try of countdown function

### PR DESCRIPTION
**Pull request by ncbecker**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,8 +1,37 @@
...
+      createElement("div", {
+        className: "time",
+        innerText: "60",
+      }),
+      createElement("button", {
+        classname: "startTimeButton",
+        innerText: "Go!",
+        onclick: () => {
+          createCountdownTimer();
+        },
```
Comment (lmachens) [inline src/App.js:17]: You don't need to create a new function

```suggestion
        onclick: createCountdownTimer,
```

File: src/App.js [hunk 2]
```diff
@@ -1,8 +1,37 @@
...
+    ],
+  });
+
+  // document.createElement("div");
+  // app.className = "app";
+  // app.innerHTML = "60";
+
+  function createCountdownTimer() {
+    const sec = 30;
+    const timer = setInterval(function () {
```
Comment (lmachens) [inline src/App.js:28]: `setInterval` returns an intervalId, `setTimeout` returns a timeoutId.
I would rename the variable.
```suggestion
    const intervalId = setInterval(function () {
```

**Review by lmachens — COMMENTED** [review]
gj 🎉

**Comment by ncbecker** [comment]
@lmachens thanks for your comments and suggestions 👍👍! Will include them and update my PR

**Pull request merged**